### PR TITLE
added Step_definition for createAccount,modified config.properties,Cr…

### PR DESCRIPTION
**Pull request by Songok2012**
added Step_definition for createAccount,modified config.properties, CreateAccCommonSteps and MyLocatorsAndMethods

### CODE REVIEW

File: aug2024_cloud/src/test/java/Step_definition/CreateAccount.java
Comment (Mabs2024) [inline aug2024_cloud/src/test/java/Step_definition/CreateAccount.java:1]: Looks good. 
I suggest deleting the steps that have been commented out from the code.

File: aug2024_cloud/src/test/java/Step_definition/CreateAccount.java [hunk 1]
```diff
@@ -1,4 +1,71 @@
...
+        //driver.findElement(By.xpath("//input[@id=\"password\"]")).sendKeys("Lenovo750Y");
+        //driver.findElement(By.xpath("//input[@id=\"password-confirmation\"]")).sendKeys("Lenovo750Y");
+
+
+    }
+
+    @When("^I select the register button$")
+    public void iSelectTheRegisterButton() throws InterruptedException {
+        MyLocatorsAndMethods pages= PageFactory.initElements(driver, MyLocatorsAndMethods.class);
+        //driver.findElement(By.xpath( "//button[@title=\"Create an Account\"]")).click();
```
Comment (Mabs2024) [inline aug2024_cloud/src/test/java/Step_definition/CreateAccount.java:54]: This line (54) can be deleted.

**Comment by Songok2012** [comment]
Okay, thanks!

File: aug2024_cloud/src/test/java/Step_definition/CreateAccount.java [hunk 2]
```diff
@@ -1,4 +1,71 @@
...
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class CreateAccount extends TestBase {
+    public int random;
```
Comment (Mabs2024) [inline aug2024_cloud/src/test/java/Step_definition/CreateAccount.java:18]: I am not sure, but you might not need line 18.

**Review by Mabs2024 — COMMENTED** [review]
Good work. If comments are addressed, I believe it should be good to go.

File: aug2024_cloud/src/test/java/Step_definition/CreateAccount.java (hunk 1 above)
Comment (Songok2012) [reply aug2024_cloud/src/test/java/Step_definition/CreateAccount.java:54]: Noted!

File: aug2024_cloud/src/test/java/Step_definition/CreateAccount.java (hunk 2 above)
Comment (Songok2012) [reply aug2024_cloud/src/test/java/Step_definition/CreateAccount.java:18]: If I get rid of it gives error. So, let me leave it.

**Review by Naskofi — APPROVED** [review]

**Pull request merged**